express-intro/server.js

```diff
@@ -0,0 +1,7 @@
+import Express from 'express';
+
+const app = Express();
+app.listen(8080, () => {
+  // eslint-disable-next-line no-console
```

Comment:
Are you getting a lint warning? If so, open your .eslintrc and remove the no-console rule (or change it to "off").

Reply:
will do